uxarray/_mpas.py

```diff
@@ -0,0 +1,266 @@
+    out_ds : xarray.Dataset
+        Output dataset where the MPAS Dual-Mesh is encoded in the UGRID
+        conventions
+    """
+
+    # corners of dual-mesh cells (in degrees)
+    lonCell = np.rad2deg(in_ds['lonCell'].values)
+    latCell = np.rad2deg(in_ds['latCell'].values)
+
+    out_ds['Mesh2_node_x'] = xr.DataArray(
+        lonCell,
+        dims=["nMesh2_node"],
+        attrs={
+            "standard_name": "longitude",
+            "long_name": "longitude of mesh nodes",
+            "units": "degrees_east",
+        })
+
+    out_ds['Mesh2_node_y'] = xr.DataArray(
+        latCell,
+        dims=["nMesh2_node"],
+        attrs={
+            "standard_name": "latitude",
+            "long_name": "latitude of mesh nodes",
+            "units": "degrees_north",
+        })
+
+    # centers of dual-mesh cells (in degrees)
+    lonVertex = np.rad2deg(in_ds['lonVertex'].values)
+    latVertex = np.rad2deg(in_ds['latVertex'].values)
+
+    out_ds['Mesh2_face_x'] = xr.DataArray(
+        lonVertex,
+        dims=["nMesh2_face"],
+        attrs={
+            "standard_name": "longitude",
+            "long_name": "longitude of center nodes",
+            "units": "degrees_east",
+        })
+
+    out_ds['Mesh2_face_y'] = xr.DataArray(
+        latVertex,
+        dims=["nMesh2_face"],
+        attrs={
+            "standard_name": "latitude",
+            "long_name": "latitude of center nodes",
+            "units": "degrees_north",
+        })
+
+    # vertex indices that surround each dual-mesh cell
+    cellsOnVertex = in_ds['cellsOnVertex'].values
+
+    # convert to zero-indexed
+    if cellsOnVertex.min() == 1:
+        cellsOnVertex -= 1
+
+    out_ds["Mesh2_face_nodes"] = xr.DataArray(
+        data=cellsOnVertex,
+        dims=["nMesh2_face", "Three"],
+        attrs={
+            "cf_role": "face_node_connectivity",
+            "start_index": int_dtype(0)
+        })
+
+    # vertex indices that saddle a given edge
+    cellsOnEdge = in_ds['cellsOnEdge'].values
+
+    # convert to zero-indexed
+    if cellsOnEdge.min() == 1:
+        cellsOnEdge -= 1
+
+    out_ds["Mesh2_edge_nodes"] = xr.DataArray(
+        data=cellsOnEdge,
+        dims=["nMesh2_edge", "Two"],
+        attrs={
+            "cf_role": "edge_node_connectivity",
+            "start_index": int_dtype(0)
+        })
+
+
+def _add_fill_values(verticesOnCell, nEdgesOnCell):
+    """Corrects the input verticesOnCell to be zero-indexed and to use fill-
+    values instead of padding / zero-fills.
+
+    Parameters
+    ----------
+    verticesOnCell : np.ndarray
+        Vertex indices that surround a given cell
+
+    nEdgesOnCell : np.ndarray
+        Number of edges on a given cell
+    """
+
+    # convert to unsigned integers
+    verticesOnCell = verticesOnCell.astype(int_dtype)
+
+    # max vertices/edges per cell
+    maxEdges = verticesOnCell.shape[1]
+
+    # is verticesOnCell one-indexed
+    one_idx = [True if verticesOnCell.min() == 1 else False]
```

Comment:
```suggestion
    one_idx = (verticesOnCell.min() == 1)
```
This is currently working for completely the wrong reasons.  It is a one-item list, whereas it is intended to be a boolean.  Then, `if one_idx` evaluates to True later on because the list is not empty.